pythonforandroid/recipes/cherrypy/__init__.py

```diff
@@ -4,7 +4,7 @@
 class CherryPyRecipe(PythonRecipe):
     version = '5.1.0'
     url = 'https://bitbucket.org/cherrypy/cherrypy/get/{version}.tar.gz'
-    depends = ['hostpython2', 'setuptools']
+    depends = [('hostpython2', 'hostpython3'), 'setuptools']
```

Comment:
Could you just remove the hostpython2 and hostpython3? They're actually unnecessary, if you don't specify anything then all python versions will be considered compatible.

Reply:
I'm resolving since it got addressed